src/dispatch/evergreen/scheduled.py

```diff
@@ -54,15 +54,24 @@ def create_evergreen_reminder(
     notification_template = EVERGREEN_REMINDER
     notification_type = "evergreen-reminder"
     name = subject = notification_text = "Evergreen Reminder"
-    success = plugin.instance.send(
-        owner_email,
-        notification_text,
-        notification_template,
-        notification_type,
-        name=name,
-        subject=subject,
-        items=items,  # plugin expect dicts
-    )
+
+    # Gmail API seems to constantly throw "tenacity.RetryError: RetryError", even though the email(s)
```

Comment:
Interesting... We are not experiencing the same behavior, but I'll take a look at our server logs to double check. Any useful errors in your GCP project that could shed light at what is happening?